### PR TITLE
Added parameters to GoogleMapPlotter

### PR DESCRIPTION
**Pull request by anschweitzer**
Added mapTypeId, scaleControl and tilt parameters to GoogleMapPlotter object. Defaults produce behavior of previous commit.

### CODE REVIEW

**Comment by anschweitzer** [comment]
#Test code for pull request: 

import os
import sys
from gmplot import gmplot

coords = [
    (42.35707236124311, -71.09224102454777),
    (42.357119930331166,-71.09210154967872),
    (42.357230924729684,-71.09175822692514),
    (42.35748858954291, -71.09103939490892),
    (42.35794841950617, -71.08963391738556),
]

param_sets = [
    ('defaults',dict()),
    ('defaults_satellite', dict(map_type='SATELLITE')),
    ('defaults_scale', dict(add_scale=True)),
    ('satellite_tilt0', dict(map_type='SATELLITE', tilt=0)),
    ('satellite_tilt0_scale', dict(map_type='SATELLITE', tilt=0, add_scale=True)),
]

if __name__ == '__main__':
    if len(sys.argv) < 2:
        raise ValueError('ERROR. Specify output directory on command line')
    out_dir = sys.argv[1]
    if not os.path.exists(out_dir):
        raise ValueError('ERROR. Output directory [%s] does not exist' % out_dir)
    latitutes = [entry[0] for entry in coords]
    longitudes = [entry[1] for entry in coords]
    for param_set in param_sets:
        out_file = os.path.join(out_dir, '%s.html' % param_set[0])
        gmap = gmplot.GoogleMapPlotter(coords[2][0], coords[2][1], 19, **param_set[1])
        gmap.plot(latitutes, longitudes, 'r')
        out_file = os.path.join(out_dir, '%s.html' % param_set[0])
        print('Saving [%s]' % out_file)
        gmap.draw(out_file)

File: gmplot/gmplot.py [hunk 1]
```diff
@@ -15,7 +15,7 @@ def safe_iter(var):
 
 class GoogleMapPlotter(object):
 
-    def __init__(self, center_lat, center_lng, zoom):
+    def __init__(self, center_lat, center_lng, zoom, map_type = 'ROADMAP', tilt = None, add_scale = False):
```
Comment (frslm) [inline gmplot/gmplot.py:18]: Let's add those parameters as keyworded arguments instead:
`def __init__(self, center_lat, center_lng, zoom, **kwargs):`

File: gmplot/gmplot.py [hunk 2]
```diff
@@ -268,10 +271,14 @@ def write_map(self,  f):
...
         f.write('\t\t\tcenter: centerlatlng,\n')
-        f.write('\t\t\tmapTypeId: google.maps.MapTypeId.ROADMAP\n')
+        f.write('\t\t\tmapTypeId: google.maps.MapTypeId.%s,\n' % self.map_type)
+        if self.add_scale:
+            f.write('\t\t\tscaleControl: true\n')
         f.write('\t\t};\n')
         f.write(
             '\t\tvar map = new google.maps.Map(document.getElementById("map_canvas"), myOptions);\n')
+        if self.tilt is not None:
+            f.write('\t\tmap.setTilt(%d)\n' % self.tilt)
```
Comment (frslm) [inline gmplot/gmplot.py:281]: We can avoid the extra `setTilt()` call and directly set `tilt` like we've done for `scaleControl`.

**Comment by frslm** [comment]
I've rebased this and addressed the comments in #151, which has been merged.

**Pull request closed without merging**